printer_mods/README.md

```diff
@@ -114,6 +114,8 @@ like so:
 | FreeformFez | [V1.8 Filtered Power Inlet](./FreeformFez/FilteredPower) | V1.6 Filtered Inlet and Rocker Switch for V1.8 | :x: | :heavy_check_mark: | :x: |:x:|
 | GadgetAngel | [Cricut Voron Logos](./GadgetAngel/Cricut_Voron_Logos) |For any cutting machines or machines that rely on vector graphics and use .svg files. These vectors or .svg files are the Voron Design Logos for the Voron 2.4, Voron SwitchWire and Voron 0.0/0.1. These .svg files use a minimal amount of anchor points to achieve the vector shapes! | :heavy_check_mark: | :x: | :heavy_check_mark: | :heavy_check_mark: | :x: |
 |  | [Bed Support Extrusions Alignment Tools](./GadgetAngel/Bed_Support_Alignment_Tools) |The bed support alignment tools will assist in helping align the "bed support extrusions" to the frame. When the alignment tools are placed in all four corners of the Voron Printer's frame the "bed support extrusions" will butt up against these tools so that you can screw the "bed support extrusions" down to the printer's frame in the correct position.|  :x: | :x: | :heavy_check_mark: |  :x: | :x: |
+|  | [Bigtreetech Octopus Pro V1.0 Color PIN Diagram](./GadgetAngel/Bigtreetech_Octopus_Pro_V1.0_Color_PIN_Diagram) |A color PIN Diagram in PDF file format (and a JPG file format) for the Octopus Pro V1.0 board that has been colorized so that the user can easily identify which endstops belong to which motors. Also, this PDF file labels all connectors instead of using "J" names. URL links have also been embedded in the PDF file so that the user can find additional information.  | :x: | :heavy_check_mark: | :heavy_check_mark: | :x: | :heavy_check_mark: |
```

Comment:
These two lines about Octopus diagrams should not be part of this PR.

Reply:
Sorry about that I forgot to delete those after I deleted the Color PIN Diagrams.  Thanks for catching that!  I fixed it and pushed a new update.